.github/workflows/build.yml

```diff
@@ -107,8 +107,7 @@ jobs:
           mkdir build
           cd build
           cmake ..
-          cmake --build . --config Release
-          ctest -C Release --output-on-failure
+          cmake --build . --config Release && ctest -C Release --output-on-failure
```

Comment:
I think it would be better for the test to be its own task instead.

Reply:
You're absolutely right. Separated build and test phases for all platforms.